matplotlib/cfg_notests.patch

```diff
@@ -0,0 +1,13 @@
+diff --git setup.cfg.template setup.cfg.template
```

Comment:
This patch is not referenced in meta.yaml, and thus will not be applied.  Did you mean to include it?